pkg/tracing/tracing.go

```diff
@@ -150,12 +153,32 @@ func createTracerProvider(service string, cfg *config.Tracing, user, pass string
 	if !cfg.Enabled {
 		return noop.NewTracerProvider(), nil
 	}
+	u, err := url.Parse(cfg.Endpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	opts := []otlptracehttp.Option{
+		otlptracehttp.WithEndpoint(u.Host),
+		otlptracehttp.WithURLPath(u.Path),
+	}
+
+	if u.Scheme == "http" {
+		opts = append(opts, otlptracehttp.WithInsecure())
+	}
+
+	if user != "" && pass != "" {
+		creds := fmt.Sprintf("%s:%s", user, pass)
+		enc := base64.StdEncoding.EncodeToString([]byte(creds))
+		o := otlptracehttp.WithHeaders(map[string]string{
+			"Authorization": fmt.Sprintf("Basic %s", enc),
+		})
+		opts = append(opts, o)
+	}
```

Comment:
Is it an option to have a user with empty password?

Reply:
What I found during testing with jaeger and nginx is that if the username or password is configured to be empty, the server just ignores the whole authorization header. So no need to pass the header in code.